framework/contracts/account/proxy/src/queries.rs

```diff
@@ -68,36 +26,6 @@ pub fn query_config(deps: Deps) -> StdResult<ConfigResponse> {
     Ok(resp)
 }
 
-/// Returns the value of the amount of the specified asset
-/// @param amount: The amount of the asset to compute the value of. If None, balance of the proxy account is used.
-pub fn query_token_value(
-    deps: Deps,
-    env: Env,
-    asset_entry: AssetEntry,
-) -> ProxyResult<TokenValueResponse> {
-    let oracle = Oracle::new();
-    let ans_host = ANS_HOST.load(deps.storage)?;
-    let asset_info = asset_entry.resolve(&deps.querier, &ans_host)?;
-    let balance = asset_info.query_balance(&deps.querier, env.contract.address)?;
-    let value = oracle.asset_value(deps, Asset::new(asset_info, balance))?;
-
-    Ok(TokenValueResponse { value })
-}
-
-/// Computes the total value locked in this contract
-pub fn query_total_value(deps: Deps, env: Env) -> ProxyResult<AccountValue> {
-    let mut oracle = Oracle::new();
-    oracle
-        .account_value(deps, &env.contract.address)
-        .map_err(Into::into)
-}
-
-pub fn query_base_asset(deps: Deps) -> ProxyResult<BaseAssetResponse> {
-    let oracle = Oracle::new();
-    let base_asset = oracle.base_asset(deps)?;
-    Ok(BaseAssetResponse { base_asset })
-}
-
 pub fn query_holding_amount(
```

Comment:
Should we remove this query as well? 

IMO this can be done by either the FE or the in the abstract-sdk bank API. Doing a seperate smart-query to this contract for this information is a bit weird.

Reply:
Agree! Removing it!

Reply:
Proxy don't need ANS_HOST anymore, should we remove it from state?

Reply:
> Proxy don't need ANS_HOST anymore, should we remove it from state?

Yup! (we should add that to the migrate logic as well then)

Reply:
we should probably also remove the migrate logic from the manager that was required for the last version?

Reply:
Tbh, now I REALLY wonder that if we merge manager and proxy, even manager will become smaller(because it have to Serialize ProxyMsg for Adding/Removing modules)

Reply:
it's pretty easy to figure out, 

- create new contract
- define the new msgs as 
```rust
ExecuteMsg {
   Proxy(proxy::ExecuteMsg),
   Manager(...),
}
```

And straight up delegate the logic to those in the match statement: 

```rust
fn execute() {
   match msg {
      Proxy(msg) => proxy::contract::execute(msg,...)
      ...
   }
}
```

and then compile it!